app/helpers/federation_helper.rb

```diff
@@ -95,10 +95,15 @@ def request_portals_names(counts, time)
     "#{output} in #{sprintf("%.2f", time)}s"
   end
 
-  def federation_enabled?
+  def federation_portals_enabled?
```

Comment:
rename to `federated_request?` as it relates to the current request state, to check if it is federated or not.

Reply:
Done https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/780/commits/745b0e13120a2cf0dcfb5e274ea5c9d0621788ca